packages/lib-classifier/src/hooks/useTranscriptionReductions.js

```diff
@@ -23,8 +23,11 @@ export default function useTranscriptionReductions() {
     }
   } = useStores()
 
-  const { loaded, caesarReductions } = useCaesarReductions(workflow.caesarReducer)
+  // NOTE: useFreehandLineReductions() and useTranscriptionReductions() are called regardless of reducer key
+  if (workflow.caesarReducer !== 'alice') return
 
+  let { loaded, caesarReductions } = useCaesarReductions(workflow.caesarReducer)
```

Comment:
Why is this using `let`? These variables don’t get reassigned. 

Probably best to specify the reducer key here, now that there’s more than one key in use. 

```suggestion
  const { loaded, caesarReductions } = useCaesarReductions('alice')
```